apps/src/lib/node/ledger/mod.rs

```diff
@@ -235,8 +236,21 @@ async fn run_aux(config: config::Ledger, wasm_dir: PathBuf) {
         maybe_start_geth(&mut spawner, &config).await;
 
     // Start oracle if necessary
-    let (eth_receiver, oracle) =
-        maybe_start_ethereum_oracle(&config, abort_sender).await;
+    let (eth_receiver, oracle_control_sender, oracle) =
+        maybe_start_ethereum_oracle(&config, abort_sender);
+
+    // TODO: pass `oracle_control_sender` to the shell for initialization from
+    // storage, rather than using a hardcoded config
+    if let Some(oracle_control_sender) = oracle_control_sender {
+        if let Err(error) = oracle_control_sender
+            .send(oracle::control::Command::SendConfig {
+                config: oracle::config::Config::default(),
+            })
+            .await
+        {
+            tracing::error!(?error, "Could not configure the oracle",);
+        }
+    }
```

Comment:
this step can be done inside `maybe_start_ethereum_oracle()`, no need to pollute `run_aux()`, to avoid further chaos like we had before

Reply:
I've moved it into `maybe_start_ethereum_oracle` and linked the TODO to https://github.com/anoma/namada/issues/686 , note this block will be temporary in any case and the `oracle_control_sender` channel will be passed down to the `Shell` in a later PR - then this block can be remove